### PR TITLE
ONEAPP-3651 - Translations for SmartApps

### PR DESCRIPTION
**Pull request by juano2310**

### CODE REVIEW

File: smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy [hunk 1]
```diff
@@ -27,7 +27,7 @@ definition(
 )
 
 preferences() {
-	section("Choose thermostat... ") {
+	section("Choose thermostat...") {
```
Comment (workingmonk) [inline smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy:30]: are there translations for these lines coming later?

File: smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy [hunk 2]
```diff
@@ -36,7 +36,7 @@ preferences() {
 	section("Air conditioning setting...") {
 		input "coolingSetpoint", "decimal", title: "Degrees"
 	}
-	section("Optionally choose temperature sensor to use instead of the thermostat's... ") {
+	section("Optionally choose temperature sensor to use instead of the thermostat's...") {
```
Comment (workingmonk) [inline smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy:39]: are there translations for these lines coming later?

File: smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy (hunk 1 above)
Comment (juano2310) [reply smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy:30]: The translations are there but the space was messing the key

File: smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy (hunk 2 above)
Comment (juano2310) [reply smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy:39]: The translations are there but the space was messing the key

File: smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy (hunk 1 above)
Comment (workingmonk) [reply smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy:30]: this key is in the translations files twice

**Review by workingmonk — APPROVED** [review]

**Pull request merged**